### PR TITLE
sheldon 0.5.4

### PR DESCRIPTION
**Pull request by rossmacarthur**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/sheldon.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Sheldon < Formula
   desc "Fast, configurable, shell plugin manager"
   homepage "https://rossmacarthur.github.io/sheldon"
-  url "https://github.com/rossmacarthur/sheldon/archive/0.5.3.tar.gz"
-  sha256 "e471dd1ce97587b373313e2cd463a145db0f72573a22aae5a605f3b676258164"
+  url "https://github.com/rossmacarthur/sheldon/archive/0.5.4.tar.gz"
+  sha256 "2707f05f59ce3df7e572f8a098d6f0d675b0b351695d04ba8d159d5a7eee8562"
   # license ["Apache-2.0", "MIT"] - pending https://github.com/Homebrew/brew/pull/7953
   license "Apache-2.0"
```
Comment (Rylan12) [inline Formula/sheldon.rb:7]: Thanks for the PR! Can you update the license and remove the comment?

```ruby
license ["Apache-2.0", "MIT"]
```

Comment (rossmacarthur) [reply Formula/sheldon.rb]: Done

**Comment by chenrui333** [comment]
@rossmacarthur, thanks for the PR, can you squash the commits into single one and commit message as the PR title? Thanks!

**Review by dtrodrigues — APPROVED** [review]
Thanks @rossmacarthur for your contribution!

**Pull request closed without merging**